poker-core/src/game/players.rs

```diff
@@ -361,7 +366,7 @@ impl SeatedPlayer {
         Ok(r)
     }
 
-    pub fn new<A: Into<PlayerId>>(id: A, monies: i32, seat_index: usize) -> Self {
+    pub fn new<A: Into<PlayerId>>(id: A, monies: Currency, seat_index: usize) -> Self {
```

Comment:
If this was replaced with Into<Currency>, that could maybe save a lot of .into() effort. I suspect the same for things that currently take PlayerId when they could take Into<PlayerId>.

Reply:
Yup.